mail_tracking_mailgun/tests/test_mailgun.py

```diff
@@ -294,33 +298,28 @@ def test_event_dropped(self):
     def test_email_validity(self, mock_request):
         self.partner.email_bounced = False
         self.partner.email = 'info@tecnativa.com'
-        mock_request.get().return_value = {
+        mock_request.get.return_value = {
             'mailbox_verification': True,
         }
         self.partner.check_email_validity()
         self.assertFalse(self.partner.email_bounced)
         self.partner.email = 'xoxoxoxo@tecnativa.com'
-        mock_request.get().return_value = {
+        mock_request.get.return_value = {
             'mailbox_verification': False,
         }
-        self.partner.check_email_validity()
+        with self.assertRaises(ValidationError):
+            self.partner.check_email_validity()
         self.assertTrue(self.partner.email_bounced)
 
     @mock.patch(_packagepath + '.models.res_partner.requests')
     def test_bounced(self, mock_request):
-        mock_request.get().return_value = {
-            'status_code': 404,
-        }
+        mock_request.get.return_value.status_code = 404
         self.partner.check_email_bounced()
         self.assertFalse(self.partner.email_bounced)
-        mock_request.get().return_value = {
-            'status_code': 200,
-        }
+        mock_request.get.return_value.status_code = 200
```

Comment:
Cool! I didn't know you could do it like this...

Reply:
I didn't either :smile: